Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.4-slim
+WORKDIR /app
+COPY ./images /app/images
+COPY ./libgrabsite /app/libgrabsite
+COPY ./grab-site ./gs-dump-urls ./gs-server ./setup.py /app/
+RUN apt-get update
+RUN apt-get install -y build-essential
+RUN pip3 install ./
+RUN apt-get purge -y build-essential
+RUN apt-get autoremove -y
+RUN apt-get clean
```

Comment:
Someone tells me that each `RUN` creates a new layer, so the purge/autoremove/clean would not reduce the size of the final Docker image.  What do you think about combining the `RUN`s on lines 6-12 into one `RUN` command?

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/minimize-the-number-of-layers